directory/directory_dest.go

```diff
@@ -147,3 +183,40 @@ func (d *dirImageDestination) PutSignatures(signatures [][]byte) error {
 func (d *dirImageDestination) Commit() error {
 	return nil
 }
+
+// returns true if path exists
+func dirExists(path string) bool {
+	_, err := os.Stat(path)
+	if err == nil {
```

Comment:
Simpler to write

return err == nil

Reply:
I think you should enhance this check.  Or rename it to pathExists,
You should handle the error, because it might be a permission denied to even see if the path exists.

Reply:
fixed